[DOC] Quantization Scheme (#25295)
openvinotoolkit/openvino: docs/articles_en/documentation/openvino-extensibility/openvino-plugin-library/advanced-guides/low-precision-transformations/quantization-scheme.rst

> .. {#openvino_docs_OV_UG_lpt_step2_markup}

Author: `{#openvino_docs_OV_UG_quantization_scheme}` is used. I'm not familiar with the format, is it page name?

> .. {#openvino_docs_OV_UG_lpt_step2_markup}

Reviewer: it's unnecessary now that we transitioned all docs to sphinx, now path is used instead of these tags, just leftovers

> .. {#openvino_docs_OV_UG_lpt_step2_markup}

Author: done, thanks

Quantization Scheme
==============================

.. meta::
   :description: Learn about quantization scheme.

.. toctree::
   :maxdepth: 1
   :caption: Low Precision Transformations

Key steps in quantization scheme:
* Low Precision Transformations: ``FakeQuantize`` decomposition to Quantize with low precision output and Dequantize. You can explore details in `Quantize decomposition <../low-precision-transformations.rst>` section.
* Low Precision Transformations: move Dequantize through operations. You can explore details in `Main transformations <../step3-main.rst>` section.
* Plugin: fuse operations with Quantize and inference in low precision.

> Quantization scheme features:

Author: not clear, should I add empty line?

> Quantization scheme features:

Author: done, thanks

> * Quantization operation is expressed through the ``FakeQuantize`` operation. ``FakeQuantize`` is not just scale and shift. You can find more details here: :doc:`FakeQuantize-1 <../../../../openvino-ir-format/operation-sets/operation-specs/operation-specs/quantization/fake-quantize-1>`. Note, please, if ``FakeQuantize`` input interval is equal output interval then ``FakeQuantize`` degenerates to ``Multiply``, ``Subtract`` and ``Convert`` (scale & shift).

Author: done, thanks

> * Dequantization operation is expressed through element-wise ``Convert``, ``Subtract`` and ``Multiply`` operations. ``Convert`` and ``Subtract`` are not mandatory. The operations can be handled as typical element-wise operations, for example: fused or transformed to another.

Reviewer: ```suggestion
* Dequantization operation is expressed through element-wise ``Convert``, ``Subtract`` and ``Multiply`` operations. ``Convert`` and ``Subtract`` are optional. These operations can be handled as typical element-wise operations, for example, fused or transformed to another.
```

Author: done, thanks

> * OpenVINO plugins fuse ``Dequantize`` and ``Quantize`` operation after low precision operation and don't fuse ``Quantize`` before.

Reviewer: ```suggestion
* OpenVINO plugins fuse ``Dequantize`` and ``Quantize`` operations after a low precision operation and do not fuse ``Quantize`` before it.
```

Quantization Scheme
==============================

.. meta::
   :description: Learn about quantization scheme.

.. toctree::
   :maxdepth: 1
   :caption: Low Precision Transformations

Key steps in the quantization scheme:

* Low Precision Transformations: ``FakeQuantize`` decomposition to Quantize with a low precision output and Dequantize. For more details, refer to the :doc:`Quantize decomposition <../low-precision-transformations>` section.

> * Low Precision Transformations: move Dequantize through operations. For more details, refer to the :doc:`Main transformations <../step3-main>` section.

Reviewer 2: ```suggestion
* Low Precision Transformations: move Dequantize through operations. For more details, refer to the :doc:`Main transformations <./step3-main>` section.
```

* Plugin: fuse operations with Quantize and inference in low precision.

Quantization scheme features:

> * Quantization operation is expressed through the ``FakeQuantize`` operation, which involves more than scale and shift. For more details, see: :doc:`FakeQuantize-1 <../../../../openvino-ir-format/operation-sets/operation-specs/operation-specs/quantization/fake-quantize-1>`. If the ``FakeQuantize`` input and output intervals are the same, ``FakeQuantize`` degenerates to ``Multiply``, ``Subtract`` and ``Convert`` (scale & shift).

Reviewer 2: ```suggestion
* Quantization operation is expressed through the ``FakeQuantize`` operation, which involves more than scale and shift. For more details, see: :doc:`FakeQuantize-1 <../../../../openvino-ir-format/operation-sets/operation-specs/quantization/fake-quantize-1>`. If the ``FakeQuantize`` input and output intervals are the same, ``FakeQuantize`` degenerates to ``Multiply``, ``Subtract`` and ``Convert`` (scale & shift).
```

Author: thanks!
